docs/api.rst

```diff
@@ -25,6 +25,5 @@
 API Docs
 ========
 
-es_jsonschema
+domapping
```

Comment:
This is referring to the module name, not the project name right? Otherwise `DoMapping` might be better.